Makefile

```diff
@@ -290,6 +290,10 @@ android-clean: ##@prepare Clean Gradle state
 	git clean -dxf -f ./android/app/build; \
 	[[ -d android/.gradle ]] && cd android && ./gradlew clean
 
+xcode-clean: SHELL := /bin/sh
+xcode-clean: ##@prepare Clean xcode derivation folder
```

Comment:
Should it be in the `prepare` category? Or `other`? If `prepare` then it should be in the `General targets` section.

Truth be told both `prepare` and `other` are kinda messy. Makefile would benefit from cleaning up those two categories. But for now that's fine. Just move it to the correct section.

Reply:
> Just move it to the correct section

It is in `other` now, by "correct section" do you mean `prepare `?

Reply:
Well, I mean position in the file. It's in:
https://github.com/status-im/status-react/blob/f1e081310d94b9d34ef25e52f58f516707d61753/Makefile#L279-L281
And most `prepare` targets are in:
https://github.com/status-im/status-react/blob/f1e081310d94b9d34ef25e52f58f516707d61753/Makefile#L111-L113

Reply:
Yep, but do you imply it should be under `prepare`? it is already under `other` as I understand

Reply:
It's fine under `prepare` for now. We can do a cleanup of these categories in the future.

Reply:
done